LunrCoreTests/EnglishStopWordFilterTests.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Lunr;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace LunrCoreTests
+{
+    public class EnglishStopWordFilterTests
```

Comment:
I ended up making this it's own test fixture since it's specific to this specific stop word filter (some other stop word filter might not use a case-insensitive set in it's implementation).